examples/collection.json

```diff
@@ -0,0 +1,135 @@
+{
+  "type": "Collection",
+  "id": "usda-cdl",
+  "stac_version": "1.0.0",
+  "description": "The USDA NASS Cropland Data Layers (CDLs) include Cropland, Confidence, Cultivated, and Frequency products. The Cropland product is a crop-specific land cover data layer created annually for the contiguous United States (CONUS). A corresponding Confidence product represents the predicted confidence that is associated with an output pixel. A value of zero indicates low confidence, while a value of 100 indicates high confidence. The Cultivated product identifies cultivated and non-cultivated land cover for CONUS and is based on land cover information derived from the 2017 through 2021 Cropland products. The Frequency product identifies crop specific planting frequency and are based on land cover information derived from the 2008 through 2021 Cropland products. There are currently four individual crop frequency data layers that represent four major crops: corn, cotton, soybeans, and wheat. All products have a 30 meter resolution.",
+  "links": [
+    {
+      "rel": "root",
+      "href": "./collection.json",
+      "type": "application/json",
+      "title": "USDA Cropland Data Layers (CDLs)"
+    },
+    {
+      "rel": "about",
+      "href": "https://www.nass.usda.gov/Research_and_Science/Cropland/SARS1a.php",
+      "title": "Product Landing Page"
+    },
+    {
+      "rel": "item",
+      "href": "./basic_cropland_2019/basic_cropland_2019.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "item",
+      "href": "./basic_cropland_2019/basic_cropland_2019.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "item",
+      "href": "./ancillary_cultivated_2021/ancillary_cultivated_2021.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "item",
+      "href": "./frequency_corn_2021/frequency_corn_2021.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "item",
+      "href": "./frequency_corn_2021/frequency_corn_2021.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "item",
+      "href": "./frequency_corn_2021/frequency_corn_2021.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "item",
+      "href": "./frequency_corn_2021/frequency_corn_2021.json",
+      "type": "application/json"
+    }
```

Comment:
Looks like we're creating multiple items with the same ID, probably because we're adding one item per asset, but some items will have multiple assets. However, this is looking good enough, so I'm going in improve and we can clean up the examples in a later PR.